client/rest-high-level/src/test/java/org/elasticsearch/client/documentation/IndicesClientDocumentationIT.java

```diff
@@ -430,18 +431,57 @@ public void onFailure(Exception e) {
             });
             // end::close-index-execute-async
         }
+    }
+
+    public void testExistsAlias() throws IOException {
+        RestHighLevelClient client = highLevelClient();
 
         {
-            // tag::close-index-notfound
-            try {
-                CloseIndexRequest request = new CloseIndexRequest("does_not_exist");
-                client.indices().close(request);
-            } catch (ElasticsearchException exception) {
-                if (exception.status() == RestStatus.BAD_REQUEST) {
+            CreateIndexResponse createIndexResponse = client.indices().create(new CreateIndexRequest("index")
+                    .alias(new Alias("alias")));
+            assertTrue(createIndexResponse.isAcknowledged());
+        }
+
+        {
+            // tag::exists-alias-request
+            GetAliasesRequest request = new GetAliasesRequest();
+            GetAliasesRequest requestWithAlias = new GetAliasesRequest("alias1");
+            GetAliasesRequest requestWithAliases = new GetAliasesRequest(new String[]{"alias1", "alias2"});
+            // end::exists-alias-request
+
+            // tag::exists-alias-request-alias
+            request.aliases("alias"); // <1>
+            // end::exists-alias-request-alias
+            // tag::exists-alias-request-indices
+            request.indices("index"); // <1>
+            // end::exists-alias-request-indices
+
+            // tag::exists-alias-request-indicesOptions
+            request.indicesOptions(IndicesOptions.lenientExpandOpen()); // <1>
+            // end::exists-alias-request-indicesOptions
+
+            // tag::exists-alias-request-local
+            request.local(true); // <1>
+            // end::exists-alias-request-local
+
+            // tag::exists-alias-execute
+            boolean exists = client.indices().existsAlias(request);
+            // end::exists-alias-execute
+            assertTrue(exists);
+
+            // tag::exists-alias-execute-async
```

Comment:
Shouldn't the change made by https://github.com/elastic/elasticsearch/pull/27899 be applied to Exists Alias (and indeed Open Index and Close Index, which were presumably merged later) as well?

Reply:
yea probably, I will sync with @tlrx who has made that change. thanks for catching this.

Reply:
That would be better, yes

Reply:
I am wondering though: the original problem was that we were creating/deleting indices asynchronously, hence we would have to wait for such operation to be completed.
Do we have to wait for any other async operation like open/close index etc.? I would say no but maybe @tlrx can confirm.

Reply:
I don't think it's important for the Exist Alias API as it does not modify the cluster state or requires a lock on shards, but for the open/close ones it might be necessary yes.

Reply:
ok I will make that change to open/close separately then